.github/workflows/ci.yml

```diff
@@ -125,7 +125,7 @@ jobs:
     strategy:
       matrix:
         rust:
-          - 1.59.0
+          - 1.65.0
```

Comment:
This line (and the one below) should probably be reverted so we can test the library-specific MSRV separately from the entire workspace. See https://github.com/parasyte/pixels/pull/320#issuecomment-1355985914 for rationale.

Note that I would not know if some dependency other than `egui` is forcing the compiler version update, but you can check locally by attempting to build just the library, e.g. with `cargo +1.59.0 build` in the `pixels-ci-rust-version` repo, pointing it to your local branch. In my testing, it still compiles with 1.59.0.